phpstan.neon

```diff
@@ -14,3 +14,5 @@ parameters:
 		- ../../extensions/WikibaseCirrusSearch
 	bootstrapFiles:
 		- ../../includes/AutoLoader.php
+	excludePaths:
+		- src/Persistence/Search/Query/HasWbFacetFeature.php
```

Comment:
This is unfortunate. PHPStan cannot find CirrusSearch classes, because there's no autoloading in their [composer.json](https://github.com/wikimedia/mediawiki-extensions-CirrusSearch/blob/master/composer.json).

I verified that if I manually add
```json
	"autoload": {
		"psr-4": {
			"CirrusSearch\\": "includes/"
		}
	}
```
and update composer, then PHPStan is happy.

So basically, either we:
* have no PHPStan on code touching CirrusSearch; or
* patch CirrusSearch locally and in CI